views/components/timePicker/timePicker.scss

```diff
@@ -5,6 +5,7 @@
 
   .time-picker-hours, .time-picker-minutes {
     width: 70px;
+    min-width:3em;
   }
 
```

Comment:
Without this, the time entry fields on the timePicker component don't honour the 70px width - they grow to the govuk-frontend min width.